src/controllers/target.controller.ts

```diff
@@ -18,26 +18,33 @@ import { TargetDTO } from '@dto/targetDTO';
 import { EntityMapper } from '@clients/mapper/entityMapper.service';
 import { Conversation } from '@entities/conversation.entity';
 import { ConversationsService } from '@services/conversation.service';
+import { Roles } from '@constants/Roles';
 
 @JsonController('/targets')
 @Service()
 export class TargetController {
   constructor(private readonly targetsService: TargetsService
     , private readonly conversationsService: ConversationsService) {}
 
-  @Authorized()
+  @Authorized(Roles.User)
   @Get()
-  async index(@CurrentUser() user :number): Promise<Target[]> {
+  async listMyTargets(@CurrentUser() user :number): Promise<Target[]> {
     return this.targetsService.listTargetsByUser(user);
   }
 
+  @Authorized(Roles.Admin)
+  @Get('/all')
+  async listAllTargets(): Promise<Target[]> {
+    return this.targetsService.listTargets();
+  }
+
   // ToDo Borrar
   // @Get('/:id')
   // async show(@Param('id') id: number): Promise<Target | undefined> {
   //   return this.targetsService.showTarget(id);
   // }
```

Comment:
I think you can delete this comment, and in case you need it, you can find it in the commit history. What do you think ?

Reply:
I agree, I forgot to delete it.